cri-o.yml

```diff
@@ -110,7 +110,7 @@
       git:
         repo: https://github.com/kubernetes-incubator/cri-o
         dest: /root/src/github.com/kubernetes-incubator/cri-o
-        version: kube-1.6.x
+        version: v1.8.0
```

Comment:
This can go to 1.8.1 now :)

Reply:
At this point, shouldn't we drop the version entirely?